test/app/app.ts

```diff
@@ -1,18 +1,20 @@
 import { datadogLogs } from '@datadog/browser-logs'
 import { datadogRum } from '@datadog/browser-rum'
 
+const origin = global ? '' : window.location.origin
```

Comment:
I'd use `location` directly, because we don't know where `global` is defined.
```suggestion
const origin = typeof location === 'object' ? location.origin : ''
```